fractal-puppy/Engine
# Fix bug in Potions persistence between levels

To test:
- Load level1 and pick some potions.
- Equip potions in the slots (q,w,e,r,...) and play to the level 2
- In the level2 you should see the potions in the correct place, with the correct number and you can use them correctly.

Also test:
. If in the level2 can open and close the inventory and skills and looks correct, not this:
 
![0ba705ef5f39f1c285c3862b8f064704](https://user-images.githubusercontent.com/7702260/67633283-6e4c4180-f8ae-11e9-84eb-f8c8b56b40d3.png)
